app/api/groups/get_permissions_can_request_help_to.feature

```diff
@@ -34,18 +46,20 @@ Feature: Get permissions can_request_help_to for a group
     When I send a GET request to "/groups/@Class/permissions/@Class/@Item"
     Then the response code should be 200
     And the response at $.granted.can_request_help_to should be:
-      | id           | name              |
-      | @HelperGroup | Group HelperGroup |
+      | id           | name              | is_all_users_group |
+      | @HelperGroup | Group HelperGroup | false              |
 
-  Scenario: Should not return helper group when set and not visible by the current user
+  Scenario: Should return helper group without the name when set and not visible by the current user
     Given I am @Teacher
     And there is a group @HelperGroup
     And there are the following item permissions:
       | item  | group  | is_owner | can_request_help_to |
       | @Item | @Class | false    | @HelperGroup        |
     When I send a GET request to "/groups/@Class/permissions/@Class/@Item"
     Then the response code should be 200
-    And the response at $.granted.can_request_help_to should be "null"
+    And the response at $.granted.can_request_help_to should be:
+      | id           | name | is_all_users_group |
+      | @HelperGroup |      | false              |
```

Comment:
I do not see (I may have missed it) example of `granted` returning a null as requested in #1004:

> in granted, the can_request_help_to is unique to keep (id, is_all_users_group, name). can_request_help_to should be null if there is no value set.

Reply:
Yes, it is the case for all tests defined in `get_permissions.feature`.